app/assets/javascripts/templates/alert_messages.hbs.erb

```diff
@@ -1,4 +1,4 @@
-{{#each info}}
+{{#each infos}}
```

Comment:
make this code more DRY ! (dont repeat yourself)
it's multiple time more or less the same ... only css class changes.
maybe you could just create one handlebar template for a message and provide css class and it's messages as parameters.

rename this file to flash_messages since it's the same messages as we us in backend's flash messages